src/main/java/Weng.java

```diff
@@ -0,0 +1,160 @@
+import duke.Deadline;
+import duke.Event;
+import duke.Task;
+import duke.Todo;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+import static java.lang.Integer.parseInt;
+
+public class Weng {
+    private static Task[] tasks = new Task[100];
+    private static int numItem = 0;
+
+    public static void main(String[] args) {
+        greeting();
+        inputHandler();
+    }
+
+    public static Task[] getTasks() {
+        return tasks;
+    }
+
+    public static void setTasks(Task[] tasks) {
+        Weng.tasks = tasks;
+    }
+
+    public static void addTodo(String[] currLine) {
+        String description = String.join(" ", Arrays.copyOfRange(currLine, 1, currLine.length));
+        addAndPrintHandler(new Todo(description));
+    }
+
+    public static void addEvent(String[] input) {
+        addAndPrintHandler(new Event(input));
+    }
+
+    public static void addDeadline(String[] input) {
+
+        addAndPrintHandler(new Deadline(input));
+    }
+
+    public static void setIndexInTask(Task task, int index) {
+        Weng.tasks[index] = task;
+    }
+
+    public static int getNumItem() {
```

Comment:
what is NumItem doing? i dont understand - can you rename this to enhance readability?